patches/third_party-blink-renderer-bindings-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/bindings/BUILD.gn b/third_party/blink/renderer/bindings/BUILD.gn
+index c468bf2c8fa437e6f8152c2a530cd354b593c20a..312c7620415baf56e8182f30357f7775aeef3d18 100644
+--- a/third_party/blink/renderer/bindings/BUILD.gn
++++ b/third_party/blink/renderer/bindings/BUILD.gn
+@@ -173,6 +173,7 @@ template("generate_bindings") {
+     args += invoker.targets
+ 
+     deps = [ ":web_idl_database" ]
++    import("//brave/third_party/blink/renderer/bindings/sources.gni") deps += brave_third_party_blink_renderer_bindings_generate_bindings_deps
```

Comment:
don't we need to add this to brave_blink_renderer_core_public_deps and shouldn't that also be sufficient to cover this? Or move brave_blink_renderer_core_public_deps to core_common or something? It seems very odd to add a cpp buildflag dependency to generated files that normally have their deps added in the targets that actually build them

Reply:
> don't we need to add this to brave_blink_renderer_core_public_deps and shouldn't that also be sufficient to cover this? Or move brave_blink_renderer_core_public_deps to core_common or something? It seems very odd to add a cpp buildflag dependency to generated files that normally have their deps added in the targets that actually build them

this exact dep should be here, because we parse PageGraph-specific args from `args.gn` from the python script that generates bindings. So we need to rerun this step exactly (not `core/public` or `core/common`) if a buildflag has changed.

If you know some other way to wire gn args with a target that doesn't use them explicitly, I would be happy to implement it.